### PR TITLE
Fix JRUBY_OPTS

### PR DESCRIPTION
**Pull request by BanzaiMan**
Resolves https://github.com/travis-ci/travis-ci/issues/5477

### CODE REVIEW

File: cookbooks/travis_build_environment/files/default/travis_environment_ruby.sh [hunk 1]
```diff
@@ -18,6 +18,6 @@ JRUBY_OPTS="$JRUBY_OPTS -Xcext.enabled=false"
 JRUBY_OPTS="$JRUBY_OPTS -J-Xss2m"
 
 # Disable invokedynamic as it is still somewhat experimental
-JRUBY_OPTS="$JRUBY_OPTS --Xcompile.invokedynamic=false"
+JRUBY_OPTS="$JRUBY_OPTS -Xcompile.invokedynamic=false"
```
Comment (bf4) [inline cookbooks/travis_build_environment/files/default/travis_environment_ruby.sh]: this should just be merged, no?

**Comment by mvz** [comment]
How can I help to get this merged? I would really like to test my gems with JRuby on Trusty :crying_cat_face:.

**Comment by igorwwwwwwwwwwwwwwwwwwww** [comment]
I've merged this fix. Please note that it will only be available on travis-ci once we update our build images, which may take some time. There is no ETA for that yet.

**Comment by bf4** [comment]
@BanzaiMan great, and next we https://github.com/travis-ci/travis-ci/issues/5733 ?

**Comment by BanzaiMan** [comment]
@bf4 Sorry, but you'd still need to work around this issue until the new build images are provisioned and rolled out. As @igorwwwwwwwwwwwwwwwwwwww said, we don't have ETA for that yet.

**Pull request merged**